sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/ExternalTypescriptSensor.java

```diff
@@ -133,17 +133,20 @@ private void analyze(
       for (SensorContextUtils.AnalysisResponse response : responses) {
         FileSystem fileSystem = sensorContext.fileSystem();
         InputFile inputFile = fileSystem.inputFile(fileSystem.predicates().hasAbsolutePath(response.filepath));
-        if (inputFile != null) {
+        if (inputFile == null) {
+          LOG.error("Failed to find input file for path `" + response.filepath + "`");
+          break;
+        }
+        if (!response.hasDiagnostics()) {
           saveHighlights(sensorContext, response.highlights, inputFile);
           saveSymbols(sensorContext, response.symbols, inputFile);
           saveMetrics(sensorContext, response, inputFile);
           saveCpd(sensorContext, response.cpdTokens, inputFile);
           SensorContextUtils.saveIssues(sensorContext, response.issues, typeScriptRules);
         } else {
-          LOG.error("Failed to find input file for path `" + response.filepath + "`");
+          SensorContextUtils.reportAnalysisErrors(sensorContext, response, inputFile);
```

Comment:
@saberduck it will be interesting to learn if on platform side they plan to use this info

Reply:
indeed, I will try to ask somebody